pom.xml

```diff
@@ -287,6 +287,13 @@
             <version>15.0</version>
         </dependency>
```

Comment:
After the pull request for the SS_SharedCode repository is merged into develop branch, we will release/version it.  We will then make sure proctor and student are dependent on that released version in their respective develop branch.